django/contrib/admin/checks.py

```diff
@@ -763,11 +781,11 @@ def _check_list_filter_item(self, obj, item, label):
 
         if callable(item) and not isinstance(item, models.Field):
             # If item is option 3, it should be a ListFilter...
-            if not issubclass(item, ListFilter):
+            if not issubclass_skip_raise_exception(item, ListFilter):
                 return must_inherit_from(parent='ListFilter', option=label,
                                          obj=obj, id='admin.E113')
             # ...  but not a FieldListFilter.
-            elif issubclass(item, FieldListFilter):
+            elif issubclass_skip_raise_exception(item, FieldListFilter):
```

Comment:
Don't see a test failure with this reverted.

Reply:
Hey, @timgraham @carltongibson I've included tests for all other cases except this one.

I'm confused on how I should reach this point in the code. 

The problem is, I can add a function which will be callable and satisfy the first case, but to reach as `elif` I have to satisfy the first case and make the `item` as a class, which would eventually not reach the case in `elif` where `item` is callable but not a class.

I tried to write a test case for this, but I couldn't figure out on how to reach to that point in the code.

Do you have any pointers?

Reply:
It might not be possible. As far as I could tell, the original patch just did a find/replace without any analysis.

Reply:
I think you can review the rest of the patch then. Please let me know if any more changes are needed.